packages/teleport/src/lib/util.ts

```diff
@@ -48,10 +48,10 @@ export async function Sha256Digest(
 }
 
 export type TshLoginCommand = {
-  accessRequestId?: string;
-  username: string;
   authType: AuthType;
-  clusterId: string;
+  clusterId?: string;
```

Comment:
Now that `clusterId` and `accessRequestId` are optional it's a good practice if they have a default value of the same type. 

Can you define some defaults for them? An empty string should be fine, you can do this in the restructuring in the function signature.

Reply:
Good idea, latest commit provides default values for the optional parameters 😄